### PR TITLE
chore: make maintenance tasks migrations safe

### PR DESCRIPTION
**Pull request by VaibhavUpreti**
#### Describe the changes you have made in this PR -

Marking potentially dangerous migrations introduced by [maintenance_tasks](https://github.com/Shopify/maintenance_tasks) as safe in to order to prevent exception thrown by [strong_migrations](https://github.com/ankane/strong_migrations) gem.

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -42,8 +42,6 @@ jobs:
       - name: Deploy with kamal
         run: |
           kamal lock release
-          kamal accessory boot yosys2digitaljs-server
-          kamal traefik boot
```
Comment (tachyons) [inline .github/workflows/deploy.yml:46]: Unrelated

Comment (VaibhavUpreti) [reply .github/workflows/deploy.yml:46]: we can remove this as we have already booted those containers.

Comment (VaibhavUpreti) [reply .github/workflows/deploy.yml:46]: Here you can see, this is my personal server:
If we run this again then that container already exists. Hence the action will fail.

test-servre

Comment (tachyons) [reply .github/workflows/deploy.yml:46]: Sorry, merged the PR without checking this thread. Our deployment script should be smart enough to check if it is already booted or not.
